docs/policy.json.md

```diff
@@ -146,13 +146,25 @@ This requirement requires an image to be signed with an expected identity, or ac
     "keyType": "GPGKeys", /* The only currently supported value */
     "keyPath": "/path/to/local/keyring/file",
     "keyData": "base64-encoded-keyring-data",
-    "signedIdentity": identity_requirement
+    "signedIdentity": identity_requirement,
+    "referencesByDigest": refs_by_digest_requirement
 }
 ```
 <!-- Later: other keyType values -->
 
 Exactly one of `keyPath` and `keyData` must be present, containing a GPG keyring of one or more public keys.  Only signatures made by these keys are accepted.
 
+The `referencesByDigest` field controls whether an reference with a digest should be taken into consideration when checking signatures or be rejected altogheter.
```

Comment:
This seems unclear to me, the user does not have the context of our PR. What is “reference” and what is “taken into consideration”?

Perhaps something like
> The `referencesByDigest` field controls treatment of images which are referred to (pulled, named, etc.) using a digest reference (e.g. `busybox@sha256:…` instead of a tag (`busybox:latest`).